actgen/train.py

```diff
@@ -92,6 +95,9 @@ def setup(self):
         self.env = env
         self.test_env = test_env
 
+        if self.params['oracle']:
+	        self.params['dqn_train_pin_other_q_values'] = True
```

Comment:
If `dqn_train_pin_other_q_values` is already set to `False`, I'd prefer to throw an exception and tell the user they messed up, rather than silently changing their hyperparams